.gitignore

```diff
@@ -31,4 +31,4 @@ _NCrunch_*/
 bundle.js
 style.css
 
-ApiApprovalTests.PublicApi.received.txt
+*.received.txt
```

Comment:
Ah yea, the old ignore was wrong now that it includes the assembly name. My fix makes sense now :P Getting tired!